### PR TITLE
Fetch additional model info: agent version, SLA and status info.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by frankban** [comment]
QA:
It must be done in gijoe and gisf scenarios (for gisf, given the changes in this branch, guiproxy should be sufficient).

- open the Javascript console and run the following:
  `b = function() {console.log(arguments)}; app.controllerAPI.listModelsWithInfo(cb);`;
- inspect the results and check that the agent version, SLA and status informations are properly reported.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7836/

File: jujugui/static/gui/src/app/store/env/controller-api.js [hunk 1]
```diff
@@ -463,7 +463,6 @@ YUI.add('juju-controller-api', function(Y) {
       Return information about Juju models, such as their names, series, and
       provider types, by performing a ModelManager.ModelInfo Juju API request.
 
-      @method modelInfo
```
Comment (jaycee) [inline jujugui/static/gui/src/app/store/env/controller-api.js]: I don't think it matters, but why are we removing the '@method' comment statement?

Comment (frankban) [inline jujugui/static/gui/src/app/store/env/controller-api.js]: It's not required anymore, and that's a good thing, as the name of the method/function is obvious without the annotation, and removing it reduces noise and means one less thing that can go out of sync (we have many examples of methods changing name but not the docstring). So I decided to remove them when I encounter them.

**Comment by jaycee** [comment]
QA

GIjoe:

Available models were 2.0.0, which do not provide the necessary information, however the GUI properly degraded without errors.

I was unable to QA gisf b/c of errors in guiproxy.

**Comment by frankban** [comment]
Thanks for the reviews!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**